topics/metagenomics/tutorials/nanopore-16S-metagenomics/tutorial.md

```diff
@@ -0,0 +1,355 @@
+> {: .solution}
+{: .question}
+
+
+# Get data
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. Create a new history for this tutorial
+> 2. Import the files from Zenodo:
+>
+>    - Open the file {% icon galaxy-upload %} __upload__ menu 
+>    - Click on __Collection__ tab
+>    - Click of the **Paste/Fetch** button
+>    - Paste the Zenodo links and press <kbd>Start</kbd> and <kbd>Build</kbd>
+>    
+>     ```
+>     https://zenodo.org/record/4274812/files/bulk_bottom.fastq.gz
+>     https://zenodo.org/record/4274812/files/bulk_top.fastq.gz
+>     https://zenodo.org/record/4274812/files/rhizosphere_bottom.fastq.gz
+>     https://zenodo.org/record/4274812/files/rhizosphere_top.fastq.gz
+>     ```
+>     - Assign a name to the new collection: `soil collection`
+>
+>    {% include snippets/import_via_link.md %}
+>    {% include snippets/import_from_data_library.md %}
+>
+> 
+{: .hands_on}
+
+# Assess datasets quality
+
+Before starting to work on our datasets, it is necessary to assess their quality. This is an essential step if we aim to obtain a __meaningful downstream analysis__.
+
+
+> ### {% icon details %} FASTQ sequence quality format
+>
+> FASTQ format is a text-based format for storing both a biological sequence and its corresponding quality scores. Both the sequence letter and the quality score are encoded with a single ASCII character for brevity. You can find more info in the [Wikipedia article](FASTQ format).
+>
+{: .details}
+
+## Quality control using FastQC and MultiQC
+
+__FastQC__ is one of the most widely used tools to __check the quality__ of the samples generated by High Throughput Sequencing (HTS) technologies. 
+
+> ### {% icon hands_on %} Hands-on: Quality check
+>
+> 1. {% tool [FastQC](toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.72+galaxy1) %} with the following parameters:
+>    - {% icon param-collection %} *"Dataset collection"*: `soil collection` 
+>
+> 2. Rename the outputs as `FastQC unprocessed: Raw` and `FastQC unprocessed: Web` 
+> 3. {% tool [MultiQC](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.8+galaxy1) %} with the following parameters:
+>    - In *"Results"*:
+>      - *"Which tool was used generate logs?"*: `FastQC`
+>      - {% icon param-collection %} *"Dataset collection"*:
+>
+>		- `FastQC unprocessed: Raw`
+> 4. Click on the {% icon galaxy-eye %} (eye) icon and inspect the generated HTML file
+> 
+>
+{: .hands_on}
+
+__FastQC__ provides information on various parameters, such as the range of quality values across all bases at each position. __MultiQC__ allows summarizing the output of different outputs from __FastQC__.
+
+![FastQC plots length](../../images/metagenomics-nanopore/fastqc_sequence_length_distribution_plot.png "Sequence length distribution")
```

Comment:
Are the two samples supposed to have different colors in the plot, or why are there two yellow lines?

Reply:
@willemdek11 Indeed there are four samples. The color coding of the plot denotes the quality scores; green is considered high, yellow medium, and red low quality score.